NewHorizons/External/Configs/StarSystemConfig.cs

```diff
@@ -31,10 +32,29 @@ public class StarSystemConfig
         public float farClipPlaneOverride;
 
         /// <summary>
-        /// Whether this system can be warped to via the warp drive. If you set factRequiredForWarp, this will be true.
+        /// Whether this system can be warped to via the warp drive. If you set `factRequiredForWarp`, this will be true.
+        /// Does NOT effect the base SolarSystem. For that, see `canWarpHome` and `factRequiredForWarpHome`
         /// </summary>
         [DefaultValue(true)] public bool canEnterViaWarpDrive = true;
 
+        /// <summary>
+        /// The FactID that must be revealed before it can be warped to. Don't set `canEnterViaWarpDrive` to `false` if
+        /// you're using this, because it will be overwritten.
+        /// </summary>
+        public string factRequiredForWarp;
+
+        /// <summary>
+        /// Can you use the warp drive to return home to the Outer Wilds from this system? If you set `factRequiredForWarpHome`
+        /// this will be true.
+        /// </summary>
+        [DefaultValue(true)] public bool canWarpHome = true;
+
+        /// <summary>
+        /// The FactID that must be revealed for you to warp back to the main solar system from here. Don't set `canWarpHome`
+        /// to `false` if you're using this, because it will be overwritten.
+        /// </summary>
+        public string factRequiredForWarpHome;
```

Comment:
maybe change `canWarpHome` to `canExitViaWarpDrive`
and `factRequiredForWarpHome` to `factRequiredForWarpFrom`

for easier understanding you could also change `factRequiredForWarp` to `factRequiredForWarpTo`

or something similar, doesn't matter.